vnext/Microsoft.ReactNative/Fabric/platform/react/renderer/graphics/conversions.h

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include <better/map.h>
+#include <folly/dynamic.h>
+#include <glog/logging.h>
+#include <react/debug/react_native_assert.h>
+#include <react/renderer/core/RawProps.h>
+#include <react/renderer/graphics/Color.h>
+#include <react/renderer/graphics/Geometry.h>
+
+namespace facebook {
+namespace react {
+
+#pragma mark - Color
+
+inline void fromRawValue(const RawValue &value, SharedColor &result) {
+  float red = 0;
+  float green = 0;
+  float blue = 0;
+  float alpha = 0;
+
+  if (value.hasType<int>()) {
```

Comment:
seems like this is missing the actual Windows PlatformColor (windowsBrush and brush names features)?